.github/workflows/maven.yml

```diff
@@ -0,0 +1,62 @@
+name: Maven CI Build
+
+on: [push]
+
+jobs:
+  JDK6Toolchain:
+    name: Toolchain 1.6, JDK 11, OS ${{ matrix.os }}
```

Comment:
Why "Toolchain 1.6, JDK 11, OS ${{ matrix.os }}" here?
Why not "JDK 1.6, OS ${{ matrix.os }}"?

Reply:
Because that's what it is?
The main difference between the two jobs is a few lines down where `mvn` gets invoked, selecting a different profile. I hope my answer on the PR makes it clear, if not please do ask more questions!